examples/nxp/aot_neutron_compile.py

```diff
@@ -167,7 +167,7 @@ def _get_batch_size(data):
         "-c",
         "--neutron_converter_flavor",
         required=False,
-        default="SDK_25_03",
+        default="SDK_25_06",
```

Comment:
What are your thoughts on FC/BC? I.e. PTE generated using an older SDK running on a newer compiler (i.e. BC) and vice versa?